homeassistant/components/tessie/binary_sensor.py

```diff
@@ -0,0 +1,146 @@
+"""Binary Sensor platform for Tessie integration."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import TessieDataUpdateCoordinator
+from .entity import TessieEntity
+
+PARALLEL_UPDATES = 0
```

Comment:
You use a coordinator, this doesn't work with PARALLEL_UPDATES

Reply:
Removed